duckdb-rfuns-r/tests/testthat/test-generated-eq.R

```diff
@@ -0,0 +1,1882 @@
+# Generated by helper-gen.R: do not edit by hand
+test_that('<int> == <int> :: 0L == 0L', {
+    con <- local_con()
```

Comment:
Can we do two spaces of indentation everywhere?

```suggestion
  con <- local_con()
```